library/build.gradle

```diff
@@ -33,11 +33,10 @@ android {
 }
 
 dependencies {
-    compile 'com.google.android:support-v4:r7'
```

Comment:
I replaced older version with the latest one 'com.android.support:support-v4:22.1.1' and added 'extra' in travis